swig/sundials/fsundials_types.i

```diff
@@ -32,6 +32,9 @@
 #define SUNDIALS_DOUBLE_PRECISION
 #define sunbooleantype int
 
+%ignore SUN_FORMAT_E;
+%ignore SUN_FORMAT_G;
+
```

Comment:
Could these `%ignore` have undesirable consequences, e.g., if Swig ends up processing a header file that includes a print statement that uses one of these?

Reply:
The format macros are only in `src/**/*.c` files, so I don't think it should be a problem. No compilation issues for me. Any concerns @balos1?